scripts/conformance_suites/run_efm_conformance_suite.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from arelle.CntlrCmdLine import parseAndRun
+
+
+CONFORMANCE_SUITE = 'tests/resources/conformance_suites/efm_conformance_suite_2022.zip/conf'
+TAXONOMY_PACKAGE = 'tests/resources/taxonomy_packages/edgarTaxonomiesPackage-22.1.zip'
+PLUGIN = 'validate/EFM'
+FILTER = '(?!arelle:testcaseDataUnexpected)'
+
+BASE_ARGS = [
+    '--logCodeFilter', FILTER,
+    '--packages', '{}'.format(os.path.abspath(TAXONOMY_PACKAGE)),
+    '--plugins', PLUGIN,
+    '--testcaseResultsCaptureWarnings',
+    '--validate'
+]
+
+IXBRL_ARGS = [
+    '--disclosureSystem', 'efm-pramatic',
```

Comment:
pragmatic?

Reply:
There are varying levels of validation for EFM. You can see them all in this config [file](https://github.com/Arelle/Arelle/blob/master/arelle/plugin/validate/EFM/config.xml). The SEC validates at level `efm-pragmatic`, which we match in wf_arelle. `efm-all-years` is used for deprecated taxonomies which are no longer accepted by the SEC.

Reply:
I think steven is pointing out this misspelling
```suggestion
    '--disclosureSystem', 'efm-pragmatic',
```